plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -119,7 +119,10 @@ module.exports = class ServiceTree extends Tree {
    */
   findServiceByName(name) {
     return this.findItem(function(item) {
-      return item.getName() === name && item instanceof Service;
+      return (
+        (item instanceof Framework && item.getFrameworkName() === name) ||
```

Comment:
In this above example @MatApple gave, what would we ideally want as output? A list of matching services, or first match (how it is now), or prioritize matching frameworks or services?